doc/radosgw/s3/authentication.rst

```diff
@@ -8,46 +8,32 @@ user. RGW supports canned ACLs.
 
 Authentication
 --------------
-Authenticating a request requires including an access key and a Hash-based 
-Message Authentication Code (HMAC) in the request before it is sent to the 
-RGW server. RGW uses an S3-compatible authentication approach. 
+Requests are authenticated with AWS Signatures which are derived from the
+user's credentials (S3 access key and secret key).
 
-::
+Most S3 clients and AWS SDKs will generate these signatures for you, given the
+necessary credentials. When issuing raw http requests, these signatures must be
```

Comment:
HTTP please

Reply:
thanks for the review. this would need to change on main and backport from there

Reply:
Sorry, on phone missed backportness

Reply:
Will pr tomorrow

Reply:
https://github.com/ceph/ceph/pull/58530